cmake/Dependencies.cmake

```diff
@@ -34,7 +34,7 @@ set(BUILD_SHARED_LIBS OFF CACHE BOOL "Global flag to cause add_library() to crea
 
 # Fetch dependencies
 if(OPENCL_SDK_BUILD_SAMPLES)
-  foreach(DEP IN ITEMS cargs TCLAP Stb)
+  foreach(DEP IN ITEMS cargs TCLAP Stb Vulkan)
```

Comment:
I think this makes Vulkan a dependency for all of the samples, correct?  It might be best to add an `OPENCL_SDK_BUILD_VULKAN_SAMPLES` option similar to `OPENCL_SDK_BUILD_OPENGL_SAMPLES`, below, so the non-Vulkan samples will continue to work even on platforms where Vulkan may be unavailable.

Reply:
Added!